server/server.go

```diff
@@ -51,12 +50,6 @@ var _ v1alpha1.CSIDriverProviderServer = &Server{}
 
 // Mount implements provider csi-provider method
 func (s *Server) Mount(ctx context.Context, req *v1alpha1.MountRequest) (*v1alpha1.MountResponse, error) {
-	deadline, ok := ctx.Deadline()
-	if !ok {
-		klog.Warningln("Mount() called without a deadline.")
-	}
-	klog.V(5).InfoS("Mount() called", "deadline", time.Until(deadline).String())
-
```

Comment:
Why remove this? Do we want to set a default deadline if there is none inbound?

Reply:
the same information is now covered by the `LogInterceptor`

Reply:
ah and deadline is set by kubelet and propagates through the driver and to our downstream RPCs through the `ctx` variable